app/phone/src/main/java/dev/jdtech/jellyfin/MainActivity.kt

```diff
@@ -38,6 +38,12 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (appPreferences.amoledTheme) {
+            setTheme(R.style.Theme_FindroidAMOLED)
+        } else {
+            setTheme(R.style.Theme_Findroid)
+        }
```

Comment:
Else statement seems to be unnecessary so I think this can be removed.

Reply:
done